src/libraries/System.Numerics.Vectors/tests/GenericVectorUnsupportedTests.cs

```diff
@@ -344,5 +344,19 @@ public void ToVectorDoubleTest()
             Vector<bool> vector = default;
             Assert.Throws<NotSupportedException>(() => (Vector<double>)vector);
         }
+
+        [Fact]
+        public void AsFromTest()
+        {
+            Vector<bool> vector = default;
+            Assert.Throws<NotSupportedException>(() => vector.As<bool, int>());
+        }
+
+        [Fact]
+        public void AsToTest()
+        {
+            Vector<int> vector = default;
+            Assert.Throws<NotSupportedException>(() => vector.As<int, bool>());
```

Comment:
This failed for `Libraries Test Run checked coreclr Linux_musl arm64 Release failed` but none of the other runs. I wonder if there is something specific about `Alpine` or `musl` that is causing this

Reply:
I've requeued that job and will try to get a repro locally. The other failures are unrelated and will be solved by https://github.com/dotnet/runtime/pull/47380

Reply:
I have no Alpine environment. Cannot investigate it.